common/test/acceptance/fixtures/catalog.py

```diff
@@ -22,14 +22,16 @@ def install_programs(self, programs):
                 will be stubbed using data from this list.
         """
```

Comment:
This function is only for setting up programs, not program types. If you need to do this, create a separate method `install_program_types` and do your setup there.